src/reference/asciidoc/file.adoc

```diff
@@ -866,3 +872,12 @@ using the `stream` option to retrieve a file, starting with _version 4.3_, the s
 the session supporting the stream, when the file is completely consumed.
 See <<ftp-streaming>> and <<sftp-streaming>> as well as <<ftp-outbound-gateway>> and <<sftp-outbound-gateway>> for more
 information about these facilities.
+
+When using Java configuration, an additional constructor is available:
+
+[source, java]
+----
+public FileSplitter(boolean iterator, boolean markers, boolean markersJson)
+----
+
+When `markersJson` is true, the markers will be represented as a JSON string, as long as a suitable JSON implementation such as Jackson or Boon is on the classpath.
```

Comment:
DITO.
Plus will add `What's New` note and modify `FtpServerOutboundTests-context.xml` to chech that new header work well:
```
<int:service-activator input-channel="markers"
	  expression="headers['file_marker']?.equals('END') ? headers['file_remoteSession'].close() : null"/>
```